pkg/services/authz/zanzana/schema/README.md

```diff
@@ -20,11 +20,19 @@ This context holds all GroupResources in a list e.g. `{ "group_resources": ["das
 
 ## Resource level permissions
 
-Most of our resource should use the generic resource type. 
+Most of our resource should use the generic resource type.
 
-To grant a user direct access to a specific resource we store `{ “user”: “user:1”, relation: “read”, object:”resource:dashboard.grafana.app/dashboard/<name>” }` with additional context.
+To grant a user direct access to a specific resource we store `{ “user”: “user:1”, relation: “read”, object:”resource:dashboard.grafana.app/dashboards/<name>” }` with additional context.
 This context store the GroupResource. `{ "group_resource": "dashboard.grafana.app/dashboards" }`. This is required so we can filter them out for list requests.
 
+## Subresources
+
+Subresources enable more granular permissions for the resources. Example might be access to public dashboards or access to dashboard settings.
+
+To grant a user access to the subresource we store following tuple: `{ “user”: “user:1”, relation: “read”, object:”resource:dashboard.grafana.app/dashboards/<subresource>/<name>” }` with additional context `{ "group_resource": "dashboard.grafana.app/dashboards/<subresource>" }`
```

Comment:
Should we clarify that name here is the name of the resource and not sub-resource?